misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/generateHUQRCodes/ConfirmOptionScreen.jsx

```diff
@@ -29,17 +30,25 @@ const ConfirmOptionScreen = () => {
             value: optionInfo.caption,
           },
           {
-            caption: trl('activities.mfg.generateHUQRCodes.qtyTUs'),
-            value: qtyTUs,
+            caption: trl('activities.mfg.generateHUQRCodes.numberOfHUs'),
+            value: numberOfHUs,
           },
         ],
       })
     );
-  }, [qtyTUs]);
+  }, [numberOfHUs]);
 
-  const validateQtyEntered = (qtyTUsEntered) => {
-    if (qtyTUsEntered <= 0) {
-      return 'QtyTUs shall be greater than 0';
+  const validateQtyEntered = (numberOfHUsEntered) => {
+    if (numberOfHUsEntered <= 0) {
+      return 'numberOfHUs shall be greater than 0';
+    }
+
+    return null; // OK
+  };
+
+  const validateNumberOfCopies = (numberOfCopiesEntered) => {
+    if (numberOfCopiesEntered < 0) {
+      return 'numberOfCopies shall be equal or greater than 0';
```

Comment:
why not >= 1?

Reply:
well, I was thinking 0 copies means `"I don't want any copies"` so print the document only once :-? but tbh, after seeing ur code with that param defaulting to ONE, I think it's better to go with one here as well